VogtObserver3/Store.cs

```diff
@@ -9,22 +9,34 @@ class Store : IObserveable
 
         private readonly List<Observer> _observers = new List<Observer>();
 
+        private float _discount;
```

Comment:
Do you need to store the discount on the class variable?

It does make sense IF you needed to know of the discoutn later, but from what I can tell, all you're doing is calling Notify(_discount) -- so you shouldn't need to store it on the class variable.